release_notes/Service_Fabric_ReleaseNotes_80CU1.md

```diff
@@ -0,0 +1,98 @@
+    The cluster is then downgraded to a previous SF runtime version (e.g. SF 7.0).<br/>
+    The application fails to activate.<br/>
+    * A cluster upgrade from pre-SF 7.1 version to SF 7.1+ version is in progress.<br/>
+    In parallel with the SF runtime upgrade, an application with guest executables or containers is created or upgraded.<br/>
+    The SF runtime upgrade starts rolling back (due to any reason) to the pre-SF 7.1 version.<br/>
+    The application fails to activate.<br/>
+
+    To avoid issues when upgrading from a pre-SF 7.1 runtime version to an SF 7.1+ runtime version, do not create or upgrade applications with guest executables or containers while the SF runtime upgrade is in progress.<br/>
+    * The simplest mitigation, when possible, is to delete and recreate the application in SF 7.0.<br/>
+    * The other option is to upgrade the application in SF 7.0 (for example, with a version only change).<br/>
+
+    If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+
+
+## Upcoming Breaking Changes
+* .NET Core runtime LTS 2.1 runtime will go out of support from Aug 21, 2021. Service Fabric releases after that date will drop support for Service Fabric apps running with .NET Core 2.1 runtime. Service Fabric .NET SDK will take a dependency on .Net runtime 3.* features to support Service Fabric .NET Core apps.  This has no impact on Service Fabric .NET Framework SDK.
+* Support for Windows Server 2016 and Windows 10 Server 1809 will be discontinued in future Service Fabric releases. We recommend updating your cluster VMs to Windows Server 2019.
+
+
+## Service Fabric Runtime
+
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 8.0.516.9590<br>Ubuntu 16 - 8.0.515.1<br>Ubuntu 18 - 8.0.515.1804** | **Feature** | Service Fabric SDK will now support Startup Services as an alternative to Default Services specified in Application Manifest | **Brief desc**: Service Fabric SDK that will support Startup Services as an alternative to Default Services specified in Application Manifest for the Visual Studio debugging experience.<br>**Impact**: This is an additional feature and will not have any impact on existing behavior.<br>**Change**: This features provides support for StartupServices.xml as an alternative to Default Services specified in ApplicationManifest.xml for the Visual Studio debugging experience. With this implementation, DefaultServices and Service definition related parameters are moved from existing ApplicationManifest.xml to a new file called StartupServices.xml. Functionality for each function - from addition of a new service to deployment (Build/Rebuild/F5/Ctrl+F5/Publish experiences in Visual Studio) is provided to work this new feature end to end. For end user to use this, VS tooling team should release the corresponding feature in VS as well. | 
```

Comment:
Can we add some insight as to why a customer would want to switch to startup services?

Reply:
There has been some communication on this throughout the years: https://stackoverflow.com/questions/50441340/auto-creation-of-service-without-defaultservices-on-developer-machines/50445801#50445801

I would also be interested in an official documentation on how to safely dig oneself out of the hole of having default services in a production app towards a "best practice" setup. (f.ex. I think there's some setting that must be set to not accidentally delete services)

Reply:
As discussed, removed note about startup services.